pmlc/conversion/tile_to_pxa/tests/broadcast.mlir

```diff
@@ -0,0 +1,33 @@
+// RUN: pmlc-opt -convert-tile-to-pxa -canonicalize -cse -split-input-file %s | FileCheck %s
+
+!t_1x2x3x4xu64 = type tensor<1x2x3x4x!eltwise.u64>
+!t_2x1x3x4xu64 = type tensor<2x1x3x4x!eltwise.u64>
+!t_2x2x3x4xu1 = type tensor<2x2x3x4x!eltwise.u1>
+
+// CHECK-LABEL: func @broadcast_has_dim_one
+func @broadcast_has_dim_one(%arg0: !t_1x2x3x4xu64, %arg1: !t_2x1x3x4xu64) -> !t_2x2x3x4xu1 {
+    %0 = "eltwise.cmp_ge"(%arg0, %arg1) : (!t_1x2x3x4xu64, !t_2x1x3x4xu64) -> !t_2x2x3x4xu1
+    return %0 : !t_2x2x3x4xu1
+
+    // CHECK: affine.load
+    // CHECK-SAME:  %arg0[0, %arg4, %arg5, %arg6] : memref<1x2x3x4xi64>
```

Comment:
In order to make the tests less brittle, the MLIR team recommends that we use regex patterns instead of matching against `%arg0` directly.

https://mlir.llvm.org/getting_started/TestingGuide/
https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-regex-matching-syntax

So, this would be preferred:

```
// CHECK-SAME:  %{{.*}}[0, %{{.*}}, %{{.*}}, %{{.*}}] : memref<1x2x3x4xi64>
```